components/grid/selection/overview.md

```diff
@@ -10,196 +10,72 @@ position: 0
+The Grid component offers support for row and cell selection. 
 
 In this article:
 
-* [Selection Basics](#selection-basics)
-	* [Example - Enable Row Selection](#example---enable-row-selection)
-	* [Example - Select rows with checkboxes only](#example---select-rows-with-checkboxes-only)
-* [Notes](#notes)
-	* [Editing Modes](#editing-modes)
-	* [Selection in Template](#selection-in-template)
-	* [Asynchronous Operations](#asynchronous-operations)
-	* [SelectedItems Equals Comparison](#selecteditems-equals-comparison)
-	* [Handle Data Changes](#handle-data-changes)
-    * [Selection in Grid with virtualized rows](#selection-in-grid-with-virtualized-rows)
-    * [Row Drag and Drop](#row-drag-and-drop)
-
-
-## Selection Basics
-
-You can configure the selection behavior by setting `SelectionMode` to a member of the `Telerik.Blazor.GridSelectionMode` enum. The row selection can be:
-
-* `None` - (the default value) to disable row selection
-* [Single]({%slug components/grid/selection/single%})
-* [Multiple]({%slug components/grid/selection/multiple%})
-
-To select a row, click on it. To select multiple rows, hold down the `Ctrl` or `Shift` key to extend the selection.
-
-You can also use a checkbox column to select rows. To use it, add a [`GridCheckboxColumn`]({%slug components/grid/columns/checkbox%}) in the `GridColumns` collection of the grid. It works with both selection modes.  With multiple selection mode, the checkbox column offers [additional functionality]({%slug components/grid/selection/multiple%}#checkbox-selection).
-
-You can get or set the selected items through the `SelectedItems` property. It is a collection of items from the Grid's `Data`.
-
-The [single selection]({%slug components/grid/selection/single%}) and [multiple selection]({%slug components/grid/selection/multiple%}) articles provide more examples and details on using the grid features.
-
-### Example - Enable Row Selection
-
-````CSHTML
-See how the row selection modes work
-
-<select @bind=@selectionMode>
-    <option value=@GridSelectionMode.Single>Single</option>
-    <option value=@GridSelectionMode.Multiple>Multiple</option>
-</select>
-
-<TelerikGrid Data=@GridData
-             SelectionMode="@selectionMode"
-             Pageable="true">
-    <GridColumns>
-        <GridCheckboxColumn SelectAll="@( selectionMode == GridSelectionMode.Single ? false : true )" Title="Select" Width="70px" />
-        <GridColumn Field=@nameof(Employee.Name) />
-        <GridColumn Field=@nameof(Employee.Team) Title="Team" />
-    </GridColumns>
-</TelerikGrid>
-
-@code {
-    public List<Employee> GridData { get; set; }
-
-    GridSelectionMode selectionMode = GridSelectionMode.Single;
+* [Selection Type](#selection-type)
+* [Selection Mode](#selection-mode)
+* [Selection Options](#selection-options)
+* [Selected Items](#selected-items)
+* [Selection Events](#selection-events)
+* [Selection and Other Grid Features](#selection-and-other-grid-features)
```

Comment:
These link labels are too brief and the uneducated reader will not benefit from them. Consider more detailed descriptions like in http://127.0.0.1:4000/blazor-ui/common-features/data-binding/onread

There is no requirement for these links to match the headings below. We already have that in the right navigation, so it brings no added value to repeat that.